examples/FSharp/FSharp.Examples/Scenarios/HelloWorlBuilder.fs

```diff
@@ -0,0 +1,87 @@
+module BuilderExample
+
+open System.Threading.Tasks
+open NBomber.Configuration
+open NBomber.Contracts
+open NBomber.FSharp
+open FSharp.Control.Tasks.V2.ContextInsensitive
+
+
+let seconds = float >> System.TimeSpan.FromSeconds
+let minutes = float >> System.TimeSpan.FromMinutes
+let connectionPool = Unchecked.defaultof<IConnectionPoolArgs<System.Net.Http.HttpClient>>
+
+let test =
+    perftest {
+        name "test name"
+        testSuite "test suite name"
+        application ApplicationType.Console
+
+        config "path/to/config/file"
+        configInfrastructure "path/to/config/file"
+
+        reports [
+            ReportFormat.Html
+            ReportFormat.Txt
+        ]
+        noReports
+        reportFileName "see_results"
+
+        scenarios [
+            scenario "pauses" {
+                pause 1000
+                pause (seconds 1.0)
+                pause (fun _ -> 1000)
+                pause (fun _ -> seconds 1.0)
+            }
+
+            scenario "simple step overloads for different action types" {
+                stepSimple "Async<unit>" (fun _ -> async {
+                    do! Async.Sleep 1000
+                })
+                stepSimple "Async<Response>" (fun _ -> async {
+                    return Response.Ok()
+                })
+
+                stepSimple "Task" (fun _ -> Task.CompletedTask)
```

Comment:
can we just have step for all cases?

Reply:
didn't found a better way until now because of type inference problems. Overloads with different arguments number are shadowing each other

Reply:
what if we leave 2 overloads
step: name:string * execute
step: name:string * feed * pool * execute
Will it lead to the same collision?

Reply:
That is a problem. Even if I delete all other overloads and keep only with `action: Task<Response>`, the builder peeks the first defined one

<img width="860" alt="image" src="https://user-images.githubusercontent.com/259479/82824069-ee6c7100-9ea8-11ea-8256-66abc5808e13.png">

It is possible to define them with another `CustomOperationAttribute`
like `step2`, `step3`, `stepWithFeed`, `stepWithFeedAndPool`. I don't really like it.
or maybe `step' (Step.create(...))` - but then it's not better than without builder.

Reply:
But multiple overloads with different input types works. For example that would compile:

```FSharp
    member _.Step(scenario : Scenario, stepName : string,
                        action : IFeed<_> * IConnectionPoolArgs<_> * IStepContext<_,_> -> Task<Response>) =
        // just create an adapter action
        let feed = Unchecked.defaultof<_>
        let pool = Unchecked.defaultof<_>
        let action' ctx = action(feed,pool, ctx)
        Step.create(stepName, action') |> addTo scenario

// and then in usage
step "simple" (fun ctx -> Task.FromResult(Response.Ok()))
step "with pool and feed" (fun (feed, pool, ctx) -> Task.FromResult(Response.Ok()))
```